elasticsearch/templates/statefulset.yaml

```diff
@@ -241,7 +241,7 @@ spec:
 
           cleanup () {
             while true ; do
-              local master="$(http "/_cat/master?h=node")"
+              local master="$(http "/_cat/master?h=node" || echo "")"
```

Comment:
@Crazybus refactored here to make `$master` be empty string if API returns an error.